### PR TITLE
Add BERT script with per epoch output

### PR DESCRIPTION
**Pull request by Peter-Devine**
Add BERT script with per epoch output, which outputs the accuracy scores of each model at every epoch

### CODE REVIEW

File: MMTL_BERT_Eval.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+model = MetalModel(tasks, verbose=False)
+
+os.environ['METALHOME'] = RESULTS_OUTPUT
+
+trainer = MultitaskTrainer()
+trainer.config["checkpoint_config"]["checkpoint_every"] = 1
+trainer.config["checkpoint_config"]["checkpoint_best"] = True
+
+for i in range(NUM_EPOCHS):
```
Comment (Peter-Devine) [inline MMTL_BERT_Eval.py:231]: This loop is all that is different to the original MMTL_BERT.py file at present

File: MMTL_BERT_Eval.py [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    scores = trainer.train_model(
+        model, 
+        payloads, 
+        n_epochs=1, 
+        log_every=1,
+        progress_bar=False,
+    )
+    print("Epoch number {0} scores:".format(i))
```
Comment (Peter-Devine) [inline MMTL_BERT_Eval.py:239]: Added printouts of scores for each epoch
